rules/internal/utils/extension_test.go

```diff
@@ -124,6 +124,84 @@ func TestGetOperationInfoNone(t *testing.T) {
 	}
 }
 
+func TestGetOperationInfoResponseType(t *testing.T) {
+	// Set up testing permutations.
+	tests := []struct {
+		testName     string
+		ResponseType string
+		invalid      bool
+	}{
+		{"Valid", "WriteBookResponse", false},
+		{"Invalid", "Foo", true},
+	}
+	for _, test := range tests {
+		t.Run(test.testName, func(t *testing.T) {
+			fd := testutils.ParseProto3Tmpl(t, `
+				import "google/longrunning/operations.proto";
+				service Library {
+					rpc WriteBook(WriteBookRequest) returns (google.longrunning.Operation) {
+						option (google.longrunning.operation_info) = {
+							response_type: "{{ .ResponseType }}"
+							metadata_type: "WriteBookMetadata"
+						};
+					}
+				}
+				message WriteBookRequest {}
+				message WriteBookResponse {}
+			`, test)
+			typ := GetResponseType(fd.GetServices()[0].GetMethods()[0])
+			if (typ == nil) != test.invalid {
```

Comment:
Same as below, imho, `typ == nil` should be given a var name, like,

```
if typeMissing := typ == nii; typeMissing == test.valid {
...
}

```

Reply:
Done.